avm/res/service-bus/namespace/topic/main.bicep

```diff
@@ -56,19 +56,8 @@ param enablePartitioning bool = false
 @description('Optional. A value that indicates whether Express Entities are enabled. An express topic holds a message in memory temporarily before writing it to persistent storage. This property is only used if the `service-bus/namespace` sku is Premium.')
 param enableExpress bool = false
 
-@description('Optional. Authorization Rules for the Service Bus Topic.')
-param authorizationRules array = [
-  {
-    name: 'RootManageSharedAccessKey'
-    properties: {
-      rights: [
-        'Listen'
-        'Manage'
-        'Send'
-      ]
-    }
-  }
-]
+@description('Optional. Authorization Rules for the Service Bus Queue.')
+param authorizationRules array = []
```

Comment:
This PR looks like covering for #2909 am I right? Would it be possible to pull the author's branch here so that we don't miss that contributor's commit?

Reply:
I had already done this before I went on leave but was having an issue with the test env. The changes here match the changes made in queues therefore superseeding #2909